Source/WebInspectorUI/UserInterface/Views/CookiePopover.js

```diff
@@ -245,6 +268,20 @@ WI.CookiePopover = class CookiePopover extends WI.Popover
 
         toggleExpiresRow();
 
+        if (this._partitionKeyInputElement) {
+            let togglePartitionKeyInput = () => {
+                this._partitionKeyInputElement.hidden = !this._partitionedCheckboxElement.childNodes[0].checked;
```

Comment:
this is not ideal as i'd expect `this._partitionedCheckboxElement` to be equal to the checkbox, not some parent